recipes/llspy-slm/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "llspy-slm" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f279eec9c2920378ea0eb8ca097d62f9ed2e5231fe585c3b2cd5a9581d9b7099
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - slmgen = slmgen.slmwindow:main
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - numpy
+    - scipy
+    - numba
+    - pillow
+    - qtpy
+    - pyqt
+
+test:
+  imports:
+    - slmgen
+  requires:
+    - pip
+  commands:
+    - pip check
```

Comment:
Is the `slmgen` entry point testable here?

Reply:
i tried [here](https://github.com/conda-forge/staged-recipes/runs/6527959637), didn't work due to qt.